### ISSUE TITLE
Support for inline disable_error_code

### ISSUE THREAD

**Issue by DevilXD** [python/mypy#10279]
**Feature**

Unless I'm doing something wrong, this appears to be currently not supported (or broken):

```py
# mypy: disable_error_code = misc

import ...
# <rest_of_the_file>
```

**Pitch**

Most other flags, work with a comment like that. This one doesn't appear so, and I'm not sure if this should be a feature request or a bug report. Constructions I tried:

```py
# mypy: disable_error_code = misc
# mypy: disable_error_code = "misc"
# mypy: disable_error_code=misc
# mypy: disable_error_code="misc"
# mypy: disable_error_code misc  # Unrecognized option: disable_error_code misc = True
# mypy: disable_error_code "misc"  # Unrecognized option: disable_error_code misc = True
# mypy: disable_error_code: misc  # Unrecognized option: disable_error_code: misc = True
# mypy: disable_error_code: "misc"  # Unrecognized option: disable_error_code: misc = True
```

The first 4 do not produce an error, but also do not work: I'm still getting `error: Self argument missing for a non-static method (or an invalid type for self)  [misc]` on the lines where I'd want that ignored. The file has 10+ of those, so I figured it'd be better to just disable them in the entire file, instead of "littering" the code with `# type: ignore[misc]`.

Python: 3.8.8
Checked both, PyPi release of `mypy-0.812`, and the current master of `mypy-0.820+dev.2a43cb95a194fa5d43d2e094be4be8e88e48f3ef`

**Comment by hauntsaninja** [comment]
https://mypy.readthedocs.io/en/stable/error_codes.html

**Comment by JukkaL** [comment]
#9440 is related.
